lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -55,3 +55,22 @@ form.ccx-form {
         margin: 5px 0 5px 0;
     }
 }
+
+button.ccx-button-link {
+    background: none !important;
+    border: none;
+    padding: 0 !important;
```

Comment:
@clrux I thought we tried to *never* use `!important` in css rules. This file has 4 being added. Is that ok??